staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -87,15 +87,6 @@ const (
 	// Add support for distributed tracing in the API Server
 	APIServerTracing featuregate.Feature = "APIServerTracing"
 
-	// owner: @tallclair
-	// alpha: v1.7
-	// beta: v1.8
-	// GA: v1.12
-	//
-	// AdvancedAuditing enables a much more general API auditing pipeline, which includes support for
-	// pluggable output backends and an audit policy specifying how different requests should be
-	// audited.
-	AdvancedAuditing featuregate.Feature = "AdvancedAuditing"
 
```

Comment:
```suggestion
```
@wlq1212, remove line no. 90, may resolve this error. otherwise Run ./hack/update-gofmt.sh